x11-wm/awesome/files/awesome-4.0-cflag-cleanup.patch

```diff
@@ -0,0 +1,11 @@
+--- awesome-4.0/CMakeLists.txt	2016-12-26 06:17:14.000000000 +0900
++++ /tmp/CMakeLists.txt	2016-12-27 09:43:26.195787200 +0900
+@@ -98,7 +98,7 @@
+ 
+ # CFLAGS
+ set(AWESOME_C_FLAGS
+-    -O1 -std=gnu99 -ggdb3 -fno-strict-aliasing -Wall -Wextra
++    -O1 -std=gnu99 -fno-strict-aliasing -Wall -Wextra
```

Comment:
so stuff gets built with `-O1`? If so, you also need to remove this, as flags that affect code generation/optimization are generally not allowed in Gentoo to be hard coded in the build system